assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -3710,6 +3711,17 @@ public void updateEntityReferences(String toContext, Map<String, String> transve
                     nAssignment.setScaleFactor(oAssignment.getScaleFactor());
                     nAssignment.setReleaseGrades(oAssignment.getReleaseGrades());
 
+                    // If there is a LTI launch associated with this copy it over
+                    Long contentKey = oAssignment.getContentId().longValue();
```

Comment:
contentId can be null and this will produce an NPE?

Reply:
Good point